doc/releases/releases.adoc

```diff
@@ -0,0 +1,3 @@
+= Releases
+
+ss
```

Comment:
Removed in context of PR #533 (https://github.com/OpenSimulationInterface/open-simulation-interface/pull/533/commits/4d42c5b46d59b87680c300af146fa305b068d0fa).